flink-contrib/flink-statebackend-redis/src/main/java/org/apache/flink/contrib/streaming/state/AbstractRedisState.java

```diff
@@ -0,0 +1,374 @@
+/**
```

Comment:
Please use regular comments (no JavaDoc) here. Best to just c&p header from other files.